### DISCUSSION TITLE
how to log values inserted into sql statements

### DISCUSSION THREAD

**Discussion by EricWvi** [SeaQL/sea-orm#1233] [Q&A] [answered]
Hi, there.
This is a snippet from my project's log. How can it also log the values that replaces the question mark?
```
INSERT INTO
  "records" (
    "name",
    "date",
    "schedule_time",
    "actual_time",
    "use_rate",
    "project_type",
    "project_id",
    "task_id"
  )
VALUES
  (?, ?, ?, ?, ?, ?, ?, ?)
```

**Comment by tyt2y3** [comment, marked as answer]
Basically you need to turn on the `debug-print` feature flag

https://www.sea-ql.org/SeaORM/docs/install-and-config/debug-log/

@billy1624 we need a section to describe how to disable SQLx logging

**Reply by EricWvi** [reply]
Thanks for reply.
I've already turned on the `debug-print` feature flag. But by the example I realize that I did not set the log level. I thought that it would output all levels of log by default. XD
Given the debug level log, I think SQLx logging, or sql statements exactly, is of little value.

**Reply by billy1624** [reply]
Hey @EricWvi @tyt2y3, thanks for the input! I just updated the SeaORM docs and added a section on how to disable SQLx logging. https://github.com/SeaQL/seaql.github.io/commit/2e7f07498fcaeeef3526d37cbd57f3ec627332fb

**Reply by tyt2y3** [reply]
Cool thanks

**Reply by EricWvi** [reply]
> Hey @EricWvi @tyt2y3, thanks for the input! I just updated the SeaORM docs and added a section on how to disable SQLx logging. [SeaQL/seaql.github.io@2e7f074](https://github.com/SeaQL/seaql.github.io/commit/2e7f07498fcaeeef3526d37cbd57f3ec627332fb)

That's what I need. Thanks!
